EIPS/eip-5388.md

```diff
@@ -0,0 +1,151 @@
+---
+eip: 5388
+title: Token-Gated HTTP Endpoints
+description: Composable RESTful and Solidity interface to implement token-gated HTTP endpoints using data tokens.
```

Comment:
I think HTTP and RESTful can be decoupled. If this EIP mandates RESTful API, the title shall reflect it. If it only mandates HTTP, then I think we can drop the `RESTful` in the description and just keep it about HTTP endpoint?

Reply:
ok

Reply:
Cool!

I kind of like to also have a RESTFul standard, but it can be one level above HTTP, hence a separate EIP.